CameraOverlay/topics.py

```diff
@@ -27,4 +27,4 @@ class Camera(NeatEnum):
 
 if __name__ == '__main__':
 	x = PowerModel.max_speed
-	print()
+	print(x)
```

Comment:
(I've only started to realise the power of `if __name__ == '__main__':`)

We should create a `Topics` class that contains all of these classes. We should then be able to run `python3 topics.py --print Camera` and it'll print out all topics for Camera. Chuck this onto trello. 😮

Reply:
Genuine question, why would this be useful? We already have the MQTT topics documented elsewhere

Reply:
That is true. I just think it's a more elegant way of running this python script. Rather than printing solely the max speed.